compiler_opt/es/blackbox_optimizers.py

```diff
@@ -290,17 +285,14 @@ def set_state(self, state):
     self.ga_optimizer.set_state(state)
 
 
-"""
-secondorder optimizers
-"""
-r"""Experimental optimizers based on blackbox ES.
+# pylint: disable=pointless-string-statement
```

Comment:
I'd just remove the pointless string statement :)

Reply:
There are multiple "pointless string statements" disabled by this pylint statement. They are used throughout the file to denote sections of the code and provide descriptions, which I feel could be useful to keep.

Reply:
ok